iocage_lib/iocage.py

```diff
@@ -897,7 +897,21 @@ def exec(self,
             login_flags = self.get('login_flags').split()
             console_cmd = ['login', '-p'] + login_flags
 
-            ioc_exec.InteractiveExec(console_cmd, path, uuid=uuid)
+            try:
+                ioc_exec.InteractiveExec(console_cmd, path, uuid=uuid)
+            except BaseException as e:
+                ioc_common.logit(
+                    {
+                        'level': 'ERROR',
+                        'message': '*** This could have likely occurred '
+                                   'because of bad permissions for '
+                                   f'{path}/root/usr/lib, please if that is '
+                                   'the case, have this corrected.'
+                    },
+                    _callback=self.callback,
+                    silent=False
+                )
+                raise e
```

Comment:
At this point we should raise a CommandFailed, as otherwise you will get an ugly stacktrace on the CLI.

Reply:
This is just the SystemExit going on and no stacktrace ;) Should we raise `CommandFailed` ?
```
root@bsd11:~ # iocage console jail
Command: login -p -f root failed!
*** Bad ownership / permissions found for files in /zroot/iocage/jails/jail/root/usr/lib/, this likely caused failure getting console access to jail jail. Please have these corrected.
root@bsd11:~ #
```

Reply:
How does that look from the API? I may be misremembering :P

Reply:
Also if do `EXCEPTION` above, we already have initiated `SytemExit`, no point in raising `CommandFailed`, thoughts ?